bing_bert/deepspeed_train.py

```diff
@@ -477,12 +439,29 @@ def prepare_model_optimizer(args):
     # DeepSpeed initializer handles FP16, distributed, optimizer automatically.
     model.network, optimizer, _, _ = deepspeed.initialize(args=args,
                                                           model=model.network,
-                                                          model_parameters=optimizer_grouped_parameters,
-                                                          dist_init_required=False)
+                                                          model_parameters=optimizer_grouped_parameters)
 
     # Overwrite application configs with DeepSpeed config
     args.train_batch_size = model.network.train_micro_batch_size_per_gpu()

```

Comment:
can we rename train_batch_size to micro_batch_per_gpu?

Reply:
done